db/version_set.cc

```diff
@@ -4230,6 +4287,20 @@ Status VersionSet::LogAndApply(
                                new_cf_options);
 }
 
+void VersionSet::LogAndApplyWalHelper(VersionEdit* edit) {
+  assert(edit->IsWalManipulation());
+
+  edit->SetPrevLogNumber(prev_log_number_);
+  edit->SetNextFile(next_file_number_.load());
+  // The log might have data that is not visible to memtbale and hence have not
+  // updated the last_sequence_ yet. It is also possible that the log has is
```

Comment:
"the log has is", typo?

Reply:
Is this comment about `two_write_queues_`?

Reply:
I copied from `LogAndApplyCFHelper`, deleted this duplicated comment.
Yes, it explains why the last sequence is related to two write queues.